### ISSUE TITLE
pycocotools PyPI error

### ISSUE THREAD

**Issue by csterling** [waikato-ufdl/wai-annotations#1]
setup.py dependency on pycocotools from Github fails with:

```
ERROR: Packages installed from PyPI cannot depend on packages which
are not also hosted on PyPI.
wai.annotations depends on pycocotools@git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI
```

**Comment by csterling** [comment]
Proposed solution:

Fork repo and deploy to PyPI under `wai.pycocotools`.

**Comment by fracpete** [comment]
I'll fork it under waikato-ufdl and set up teams. Then you can work on it and deploy it.

**Comment by fracpete** [comment]
The fork is available here and `csterling` should have write access:
https://github.com/waikato-ufdl/cocoapi

**Comment by fracpete** [comment]
MMDetection maintains their own fork of the COCOAPI repo:
https://github.com/open-mmlab/cocoapi
Check whether this repo works with wai.annotations. If confirmed, then I will recreate the cocoapi UFDL repo, forking from that repo instead. MMDetection does not work with the other repo, as they changed functionality.

**Comment by csterling** [comment]
The [repo](https://github.com/open-mmlab/cocoapi) does seem to work

**Comment by fracpete** [comment]
And forked into https://github.com/waikato-ufdl/cocoapi

**Comment by csterling** [comment]
Implemented in commit 965768b9347f52aac4dbfeb34a7f79bab8606fa6.
